index.html

```diff
@@ -0,0 +1,471 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="index.css" />
+
+
+  <title>Portifolio mobile</title>
+
+</head>
+
+<body>
+  <header class="first-section">
+    <div class="font">
+
+      <div class="first-container">
+        <div class="font">
+
+          <div class="header">
+            <a class="my-logo"><strong>My Logo</strong></a>
+            <nav class="hamburger">
+              <ul class="ul-style">
+                <li>
+                  <a href="#">
+                    <img src="./images/Union.svg" alt="union background" />
+                  </a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="section1">
+      <h1 class="first-para">
+        I’m Mirela<br />
+        Glad to see <br />
+        you!
+      </h1>
+      <div class="second-para">
+        <img src="./images/primary_text.jpg" alt="text">
+      </div>
+      <p class="third-para">LET’S CONNECT</p>
+
+      <ul class="social_links">
+
+
+        <li><a href=""><img src="./images/Vector-2.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Linkedin-icon.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Vector-3.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Vector.jpg" class="logo-icons" alt="union" /></a></li>
+
+        <li><a href=""><img src="./images/Vector-4.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+      </ul>
+    </section>
+
+    <section class="container1">
+      <div>
+        <div class="Pics">
+
+          <img src="./images/Snapshoot-icon.jpg" alt="Snapshoot" />
+        </div>
+
+        <div class="tonic">
+          <img src="./images/tonictitle.jpg" alt="tonic">
+        </div>
```

Comment:
- [ ] Please kindly make sure the project title uses a `hx` tag and not an `img` tag. Kindly also make this change for subsequent cards